app/src/main/res/layout/fragment_tool_details.xml

```diff
@@ -31,6 +33,7 @@
                 android:layout_alignBottom="@id/banner"
```

Comment:
How does this work if the `@id/banner` element (`SimplePicassoImageView`) is not visible and the `@id/video_banner` element (`YouTubePlayerView`) is visible?